challenge-07/challenge-07.js

```diff
@@ -1,51 +1,65 @@
 /*
 Crie um array com 5 items (tipos variados).
 */
-// ?
+var arr = [1, 'Bruno', 'Tortelli', true, 27];
 
 /*
 Crie uma função chamada `addItem`, que irá adicionar itens no array criado.
 A função deverá retornar o array atualizado.
 */
-// ?
+function addItem(param){
+  arr.push(param);
+  return arr;
+};
 
 /*
 Adicione um novo array ao array criado no início do desafio, com ao menos 3
 itens de tipos diferentes, mostrando o resultado no console.
 */
-// ?
+console.log(addItem([2, 'Javascript Ninja', true]));
 
 /*
 Mostre no console o segundo elemento desse último array, criado acima, com a
 frase:
 "O segundo elemento do segundo array é [ELEMENTO]."
 */
-// ?
+console.log('O segundo elemento do segundo array é ' + arr[5][1] + '');
 
 /*
 Mostre no console quantos itens tem o primeiro array criado, com a frase:
 "O primeiro array tem [QUANTIDADE DE ITENS] itens."
 */
-// ?
+console.log('O primeiro array tem ' + arr.length + ' itens.');
 
 /*
 Agora mostre no console quantos itens tem o segundo array criado, com a frase:
 "O segundo array tem [QUANTIDADE DE ITENS] itens."
 */
-// ?
+console.log('O segundo array tem ' + arr[5].length + ' itens.');
 
 /*
 Utilizando a estrutura de repetição `while`, mostre no console todos os números
 pares entre 10 e 20, inclusive esses 2.
 */
 console.log( 'Números pares entre 10 e 20:' );
-// ?
-
+num = 10;
+while(num <= 20){
+  if(num % 2 === 0){
+    console.log(num);
+  };
```

Comment:
-1
Após funções literais, estruturas condicionais, e estruturas de repetição (for, while, etc), não se usa ponto-e-vírgula.